realm-transformer/src/main/kotlin/io/realm/transformer/RealmTransformer.kt

```diff
@@ -133,19 +137,14 @@ class RealmTransformer(val project: Project) : Transform() {
             }
 
             var containsKotlin = false
-
+            // Should be safe to iterate the configurations as we are way beyond the configuration
+            // phase
             outer@
-            for(input: TransformInput in inputs) {
-                for (di: DirectoryInput in input.directoryInputs) {
-                    val path: String = di.file.absolutePath
-                    val index: Int = path.indexOf("build${File.separator}intermediates${File.separator}classes")
-                    if (index != -1) {
-                        val projectPath: String = path.substring(0, index)
-                        val buildFile = File(projectPath + "build.gradle")
-                        if (buildFile.exists() && buildFile.readText().contains("kotlin")) {
-                            containsKotlin = true
-                            break@outer
-                        }
+            for (configuration in project.configurations) {
+                for (dependency in configuration.dependencies) {
+                    if (dependency.name.startsWith("kotlin-stdlib")) {
```

Comment:
Is the dependency name just the "middle" part? e.g. this would normally be `org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.72`, so it doesn't start with `org.` ?

Reply:
Format is `<group>:<name>:<version>`. Seems to do the right thing...as opposed to the old one which I printed after the `/`
 ```Analyzed:  name=architectureComponentsExample,  containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=compatibilityExample,           containsKotlin=true/false,   sync=true,   targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=encryptionExample,              containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=gridViewExample,                containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=introExample,                   containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=jsonExample,                    containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=kotlinExample,                  containsKotlin=true/false,   sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=migrationExample,               containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=mongoDbRealmExample,            containsKotlin=true/false,   sync=true,   targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=multiprocessExample,            containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=newsreaderExample,              containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=rxJavaExample,                  containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=threadExample,                  containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=unitTestExample,                containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
Analyzed:  name=library,                        containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=library
Analyzed:  name=app,                            containsKotlin=false/false,  sync=false,  targetSdk=29,  minSdk=16,  target=app
```

Reply:
Cool 👍